src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+
+  const convertedObj = {};
+
+  sourceString.split(';').map((elem) => {
+    const startValueIndex = elem.indexOf(':');
+
+    const parametr = elem.slice(0, startValueIndex).trim();
+    const value = elem.slice(startValueIndex + 1).trim();
+
+    parametr.replace(/\n/g, '');
```

Comment:
[CODE KNOWLEDGE] - The `replace` method does not change the original string. It returns a new string. In your case, you are not using the new string returned by `replace` method. You should assign the result of the `replace` method to `parametr` variable.